lib/liquid/profiler/hooks.rb

```diff
@@ -19,5 +19,14 @@ def render_with_profiling(context)
 
     alias_method :render_without_profiling, :render
     alias_method :render, :render_with_profiling
+
+    def render_to_output_buffer_with_profiling(context, output)
+      Profiler.profile_children(context.evaluate(@template_name_expr).to_s) do
+        render_to_output_buffer_without_profiling(context, output)
+      end
+    end
+
+    alias_method :render_to_output_buffer_without_profiling, :render_to_output_buffer
+    alias_method :render_to_output_buffer, :render_to_output_buffer_with_profiling
```

Comment:
Do we still need `render_with_profiling`, given `Include#render` is now not valid?